.ci/openvino-onnx/Jenkinsfile

```diff
@@ -84,13 +122,19 @@ pipeline {
                 gitSubmoduleUpdate(PROJECT_NAME)
             }
         }
-        stage("Run tests") {
+        stage("Prepare Docker environment") {
             steps{
                 dir("${WORKDIR}") {
                     buildDockerImage()
+                    runDockerContainer("${IMAGE_ID}")
```

Comment:
This can be simplified, you don't need to split `run` and `exec`.

Reply:
Good idea, fixed